kinode/src/net/mod.rs

```diff
@@ -1,9 +1,6 @@
 #[cfg(feature = "simulation-mode")]
 pub mod mock;
```

Comment:
does `mock` still exist? why? I thought the code there was all obsolete with this